webhook/runtime_webhooks.md

```diff
@@ -0,0 +1,103 @@
+- Come back to developer console and you will see deployed runtime action in your namespace under the `User Defined Actions` section once you refresh.
+- Now set up an event registration using the runtime action you just deployed. You would need to select option 2 (Runtime Action) in your `Configure event registration` page. Choose your runtime action from the dropdown and click `Save configured events`
+- This will create an event registration with an event handler webhook pointing to your runtime action. 
+
+
+## Benefits and Improvements using Runtime Action as Webhook
+
+If your use-case suits having a runtime action that responds within a short time frame, we recommend using the webhook (SYNC based) approach to get the below mentioned benefits.
+
+- [Built In Signature Verification](#built-in-signature-verification)
+- Improved Debugging 
+    - [Debug tracing shows your action response](#debug-tracing-shows-your-action-response)
+    - [Debug tracing shows activation ids for failed invocations](#debug-tracing-shows-activations-ids-for-failed-invocations)
+- [No more duplicate action invocations](#avoiding-duplicate-action-invocations)
+
+
+### Built In Signature Verification
+
+From the security standpoint of webhook invocation, until now there was no way to verify that the event invocation requests to your publicly exposed runtime webhooks are coming from I/O Events and not any malicious user. But, now your runtime webhooks are made secure by being provided with an OOTB signature verification implementation thats working under the hood. So, basically whatever runtime action you use to create an event registration the webhook created due to that will place a signature validator action along with your action as in a [sequence](#https://github.com/AdobeDocs/adobeio-runtime/blob/master/reference/sequences_compositions.md). I/O Events signs the event payload with client's secret signature and pass them in request header to webhook. Whatever requests will come to your webhook will only invoke your business logic runtime action when the validator action successfully verifies that the correct signature has been passed to invoke your webhook.
+
+### Debug Tracing Shows Your Action Response
+
+Debug tracing is a pretty important tool on developer console for users who want to be informed whether their webhook invocation is successful or not or what it responds. However, there was a gap in this feature wherein users could not know if their action has succeeded or what the action actually returned, until now. Users could only see an empty response body (as shown below) in the `Debug Tracing` response which was not a good user experience.  
+
+![Debug Tracing Webhook Response Old on Adobe Developer Console](../img/debug_tracing_webhook_response_before.png)
+
+Now, after setting up a runtime action as webhook, you can see your main action custom response for both success and failure cases in the `Debug Tracing` webhook response section as below.
```

Comment:
You should document the most recent change you made just before your PTO (as discussed [here](https://adobeio.slack.com/archives/C6841FMPU/p1617228375088800?thread_ts=1616536045.029100&cid=C6841FMPU)), that if the user action returns a response, that response is shown in Debug Tracing